### DISCUSSION TITLE
Context acquisition from adjacent cells in Jupyter Notebook bugged in VSCode

### DISCUSSION THREAD

**Discussion by martingarridorc** [community/community#64039] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi,

This bug has already been reported [here](https://github.com/orgs/community/discussions/62569) and [here](https://www.reddit.com/r/learnprogramming/comments/15ano7q/github_copilot_and_jupyter_notebook_suddenly/). But I could not find it in this forum which is the reason why I am posting it here.

Briefly, since the version 1.88.132, the VSCode extension lost its ability to grab context from adjacent cells in Jupyter Notebooks. It was not a problem yesterday as downgrading the Copilot extension was making the trick. But now we are forced to use the most recent version for it to work. Hence, we have lost the ability to learn from adjacent cells in Jupyter Notebook (VSCode).

This is happening to me in the Macbook Pro 2022 with M1 processor. Other details are:

```
Python version: 3.10.12
Jupyter notebook version: 7.0.2
```

**Comment by Akash1134** [comment]
Hi @martingarridorc ,

Thanks for reporting! I have shared your report with my team internally and we will get back to you with more details.

**Comment by hsenag** [comment, marked as answer]
Sorry about this. We believe it's fixed in 1.105, which has just been released. Please let us know if that's not the case!

**Reply by martingarridorc** [reply]
Perfect! Thank you very much! Once I install the new version I will reach out if something stills buggy
